articles/shared/shared-how-access-ubuntu-update-servers.md

```diff
@@ -23,37 +23,36 @@ When you create a virtual machine (VM) with an Ubuntu licence on it, you need to
 
 Before you attempt to establish connections to an Ubuntu repository server, you need to make sure your VMs can communicate with the Ubuntu server which exists outside of your cloud organisation. This may involve editing the NAT and firewall settings within your edge gateway to allow traffic to traverse into your virtual data centre (VDC). For how to do this, see the [*How to create NAT rules*](../vmware/vmw-how-create-nat-rules.md) and [*How to create firewall rules*](../vmware/vmw-how-create-firewall-rules.md).
 
-## Assured OFFICIAL platform
+### Assured OFFICIAL platform
 
 UKCloud's Assured OFFICIAL platform is internet facing, so you can either configure your VM to connect to the internet, and use standard update tools such as apt-get to talk directly to the publicly available Ubuntu repositories or you can use the UKCloud-managed repository servers.
 
-## Elevated OFFICIAL platform
+### Elevated OFFICIAL platform
 
-UKCloud's Elevated OFFICIAL platform doesn't natively connect to the internet, and the PSN Protected network doesn't have any Ubuntu repository servers. To connect to an Ubuntu repository server for updates, you can use UKCloud-managed repository servers or a UKCloud Walled Garden.
+UKCloud’s Elevated OFFICIAL platform doesn't have native access to Ubuntu repository servers. To connect to an Ubuntu repository server for updates, you can use UKCloud-managed repository servers or a UKCloud Walled Garden.
```

Comment:
Apostrophe change for UKCloud's but not doesn't, etc...?

Reply:
fixed